tests/test_client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import requests
+
+from tap_wonolo.client import is_fatal_code
+
+
+@pytest.mark.parametrize('status_code', [400, 401, 403, 404, 429,
+                                         pytest.param(500, marks=pytest.mark.xfail),
+                                         pytest.param(502, marks=pytest.mark.xfail),
+                                         pytest.param(503, marks=pytest.mark.xfail),
+                                         pytest.param(504, marks=pytest.mark.xfail)])
+def test_is_fatal_code_true(status_code):
```

Comment:
add a test for a `requests.exceptions.Timeout` and/or `requests.exceptions.ConnectionError` (where AFAIK we won't have a _response_ to inspect(?))

Reply:
@bobzoller interesting. I think that would have to be covered in a different function since the `is_fatal_code` function requires a response object